config/routes.rb

```diff
@@ -4,6 +4,10 @@
   devise_for :admins, path: 'admins'
   devise_for :employees, path: 'employees'
   resources :kudos
-    get '/admin' => "pages#dashboard", :as => :admin_root
+  namespace :admins do
+    resources :kudos
```

Comment:
we should limit this endpoint only to methods that we are using in controller (`index`,`destroy`)